packages/remark-lint-no-paragraph-content-indent/index.js

```diff
@@ -163,7 +163,7 @@ function head(node) {
 }
 
 function applicable(node) {
-  return 'value' in node || 'alt' in node;
+  return 'value' in node || ('alt' in node && node.alt !== null);
```

Comment:
...and not use  `node.alt` instead of `('alt' in node && node.alt !== null)` here?